src/Model/ImageLoading.py

```diff
@@ -123,8 +129,43 @@ class NotInteroperableWithOnkoDICOMError(Exception):
         Exception (_type_): _description_
     """
 
+def _is_correct_orientation(slice_data: FileDataset) -> bool:
+    """
+    Determines if a DICOM slice is oriented in an axial plane.
+    Checks the orientation of a DICOM slice by calculating its normal vector
+    and comparing it to the standard Z-axis. Slices within approximately
```

Comment:
This means that images that were sagittal (common for MR) or coronal (not as common, but still used in MR) can not be fused?  That's OK if it was intentional.  Need to discuss with Dr. Miller what's supposed to happen with Sagittal and Coronal series.

You could detect these in a similar fashion as you've done with Axial by choice of vector [1,0,0] and [0,1,0] to project the cross product of the Image Position Patient on to.  And for now, if those aren't supportable, to indicate that in a clear message to the user.